content/learn/getting-started.es.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Primeros pasos
+mobile_menu_title: "Primeros pasos"
+toc: true
+---
+
+## Lanzamientos etiquetados o compilación nocturna?
+Zig no ha alcanzado la versión 1.0 aún y el ciclo de liberación actual depende de la liberación de LLVM, lo cual ocurre cada ~6 meses.
```

Comment:
Might be worth considering `lanzamiento` as an alternative translation - I'm not sure that `liberación` is commonly used to refer to "software release". `ciclo de actualización` would also be a good option, referring to the update cycle instead

Reply:
"lanzamiento" is an excellent alternative, good catch